src/components/Content/Pursuance/PursuancePage.js

```diff
@@ -20,21 +21,22 @@ class PursuancePage extends Component {
 
   render() {
     return (
-      <Router>
-        <div id="pursuance-page" className="content-ctn">
-          <nav id="pursuance-nav">
-            <PursuanceMenu />
-          </nav>
-          <article>
-            <Switch>
-              <Route exact path="/pursuance/:pursuanceId" component={TaskListView} />
-              <Route exact path="/pursuance/:pursuanceId/tasks" component={TaskListView} />
-              <Route exact path="/pursuance/:pursuanceId/calendar" component={CalendarView} />
-              <Route exact path="/pursuance/:pursuanceId/discuss/task/:taskGid" component={DiscussView} />
-              <Route exact path="/pursuance/:pursuanceId/participants" component={ParticipantsView} />
-            </Switch>
-            <RightPanel />
-          </article>
+    <Router>
+        <div id="pursuance-page" className="content-ctn" onKeyDown={(event) => {this.child.handleKeyDown(event);}} tabIndex="0">
```

Comment:
`onKeyDown={(event) => {this.child.handleKeyDown(event);}}`

can be just

`onKeyDown={(event) => this.child.handleKeyDown(event)}`

or in this case -- even simpler -- just

`onKeyDown={this.child.handleKeyDown}`